docs/page-objects/testing.rst

```diff
@@ -72,6 +72,10 @@ Page Object directory from an iterable of dependencies, an output item and an
 optional metadata dictionary. It can optionally take a name for the fixture
 directory. By default it uses incrementing names "test-1", "test-2" etc.
 
+.. note::
+    ``output.json`` contains a result of
+    ``json.dumps(ItemAdapter(page_object.to_item()).asdict()``.
```

Comment:
```suggestion
    ``json.dumps(ItemAdapter(page_object.to_item()).asdict())``.
```

This is also not technically true, as we use some JSON options like ensure_ascii=True. What do you think about replacing it with something like

```
``ItemAdapter(page_object.to_item()).asdict()`` saved as JSON
```